tests/test_e2e.py

```diff
@@ -89,15 +63,16 @@ def test_task_journey(driver, test_app):
     assert driver.title == 'To-Do App'
 
     add_new_task(driver)
-    #start_task(driver)
-    #complete_task(driver)
-    #mark_test_as_incomplete(driver)
+    # start_task(driver)
```

Comment:
If we're not planning to reenable these shortly, we should get rid of them. We can always recover them with git